ballerina/document_types.bal

```diff
@@ -0,0 +1,118 @@
+# Enumeration of supported document types
+public enum DocumentKind {
+    # Text document type
+    TEXT = "text",
+    # Image document type  
+    IMAGE = "image",
+    # Audio document type
+    AUDIO = "audio",
+    # File document type
+    FILE = "file"
+}
+
+# Enumeration of MIME types for different document formats
+public enum DocumentMimeType {
+    # Plain text format
+    TEXT_PLAIN = "text/plain",
+    # HTML format
+    TEXT_HTML = "text/html",
+    # CSS format
+    TEXT_CSS = "text/css",
+    # CSV format
+    TEXT_CSV = "text/csv",
+    # JPEG image format
+    IMAGE_JPEG = "image/jpeg",
+    # PNG image format
+    IMAGE_PNG = "image/png",
+    # GIF image format
+    IMAGE_GIF = "image/gif",
+    # SVG image format
+    IMAGE_SVG = "image/svg+xml",
+    # WebP image format
+    IMAGE_WEBP = "image/webp",
+    # JSON format
+    APPLICATION_JSON = "application/json",
+    # XML format
+    APPLICATION_XML = "application/xml",
+    # PDF format
+    APPLICATION_PDF = "application/pdf",
+    # ZIP archive format
+    APPLICATION_ZIP = "application/zip",
+    # Binary data format
+    APPLICATION_OCTET_STREAM = "application/octet-stream"
+}
+
+# Enumeration of supported audio formats
+public enum AudioFormat {
+    # WAV audio format
+    WAV = "wav",
+    # MP3 audio format
+    MP3 = "mp3"
+}
+
+# Represents a URL pointing to the content of a document
+public type Url record {|
+    # The URL pointing to the content of the document
+    string url;
+|};
```

Comment:
Didn't we decide to use `string` with a constraint annotation.

Reply:
yes, Will update